src/og.rs

```diff
@@ -11,16 +11,6 @@
 use scraper::{Html, Selector};
 use std::fmt;
 
-#[derive(Debug)]
-pub struct OpenGraph {
-    title: Option<String>,
-    description: Option<String>,
-    url: Option<String>,
-    image: Option<String>,
-    object_type: Option<String>,
-    locale: Option<String>,
-}
-
```

Comment:
Not sure why is this being removed?

Reply:
As far as I can see `OpenGraph` is not used anywhere inside the project. Using it from the outside does not make any sense since all the fields are private.
But probably I have overseen something?